sqlglot/dialects/bigquery.py

```diff
@@ -898,6 +898,9 @@ def intersect_op(self, expression: exp.Intersect) -> str:
                 self.unsupported("INTERSECT without DISTINCT is not supported in BigQuery")
             return f"INTERSECT{' DISTINCT' if expression.args.get('distinct') else ' ALL'}"
 
+        def mod_sql(self, expression: exp.Mod) -> str:
```

Comment:
for simple things like this, you can just put it in transforms, i think you could do

exp.Mod: rename_func("MOD")